assets/scripts/months.js

```diff
@@ -165,8 +165,13 @@ $(document).ready(function() {
 
   	  console.log(`Our current event is: ${value}`);
 
+  	  //Events_list and colors_list are used to help set up the linking system.
       var events_list = $(".eventname").map(function(){return this.innerHTML;}).get();
-      var colors_list = $(".colorpick").map(function(){return this.innerHTML;}).get();
+      var colors_list = $(".colorpick").map(function(){
```

Comment:
Is there a way to get the .value of the color picker, as done on line 89, instead of splitting and substringing the HTML?